src/main/java/com/internship/backend/service/IdGeneratorService.java

```diff
@@ -0,0 +1,39 @@
+package com.internship.backend.service;
+
+import com.internship.backend.repository.*;
+import org.springframework.stereotype.Service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+@Service
+public class IdGeneratorService {
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Autowired
+    private AuthorityRepository authorityRepository;
+
+    @Autowired
+    private ReservationRepository reservationRepository;
+
+    @Autowired
+    private LocationRepository locationRepository;
+
+    @Autowired
+    private TennisCourtRepository tennisCourtRepository;
+
+    @Autowired
+    private PriceRepository priceRepository;
+
+    public int getCurrentId(){
```

Comment:
Does this id need to be uniq? If it does, i'm not sure if sum of these counts will be enough to handle that.